api/conf/schema.json

```diff
@@ -574,7 +574,7 @@
 						{
 							"required": [
 								"discovery_type",
-								"service_name",
+								"service_name_type",
```

Comment:
Please do not modify this file, it is generated directly by the APISIX Control API and should not be edited by a human.